.sequelizerc

```diff
@@ -5,6 +5,6 @@ const path = require('path');
 module.exports = {
   'config': path.resolve('server/database/config', 'config.js'),
   'models-path': path.resolve('server/database/models'),
-  'seeds-path': path.resolve('server/database/seeds'),
+  'seeders-path': path.resolve('server/database/seeds'),
```

Comment:
ditto

Reply:
the seeds script wasn't working. After researching, I found out I had to change the path. I also inform the database code owner.